src/hb-subset-input.cc

```diff
@@ -478,4 +480,26 @@ hb_subset_preprocess (hb_face_t *source)
 
   return new_source;
 }
+
+/**
+ * hb_subset_input_override_name_table:
+ * @input: a #hb_subset_input_t object.
+ * @name_id: name_id of a nameRecord
+ * @name_str: pointer to name string new value or null to indicate should remove
```

Comment:
Can you clarify that a null will remove the string from the override map, not from the name table.

Reply:
Oh sorry I misread the code I thought you were deleting the entry from the map when null, but instead you're removing it from the name id set which is correct. So nothing to do here.